mne_bids/commands/tests/test_cli.py

```diff
@@ -255,4 +263,37 @@ def test_count_events(tmpdir):
         mne_bids_count_events.run()
 
 
+@requires_pandas
+@requires_matplotlib
+def test_inspect(tmpdir):
+    """Test mne_bids inspect."""
+
+    # Check that help is printed
+    check_usage(mne_bids_inspect)
+
+    # Create test dataset.
+    bids_root = str(tmpdir)
+    data_path = testing.data_path()
+    subject = '01'
+    task = 'test'
+    datatype = 'meg'
+    raw_fname = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc_raw.fif')
+
+    raw = mne.io.read_raw(raw_fname)
+    raw.info['line_freq'] = 60.
+
+    bids_path = BIDSPath(subject=subject, task=task, datatype=datatype,
+                         root=bids_root)
+    write_raw_bids(raw, bids_path, overwrite=True, verbose=False)
+
+    import matplotlib
+    matplotlib.use('agg')
+
+    with ArgvSetter(('--bids_root', bids_root, '--subject', subject,
+                     '--task', task, '--type', datatype)):
+        with pytest.warns(RuntimeWarning, match='The unit for chann*'):
+            mne_bids_inspect.run()
```

Comment:
running `mne_bids inspect --bids_root BIDS` should be enough. Ie there should be default values None so I can just loop over all files with just mne_bids inspect --bids_root BIDS

Reply:
Yes, this was an oversight. Fixed in 6d5cf3c